tabdil/time.py

```diff
@@ -1,19 +1,61 @@
-
```

Comment:
Why did you remove these lines?

Reply:
To be honest, for no special reason!
The line with no code was annoying for me, therefore I just removed it.

Reply:
I didn't precisely remove any line. I just added some new lines like the comments.
But the way the GitHub shows the differences made you think that I removed anything.